package.json

```
@@ -42,29 +42,29 @@
         "watch": "mocha \"tests/lib/**/*.js\" --reporter dot --watch --growl"
     },
     "dependencies": {
-        "@typescript-eslint/eslint-plugin": "~5.60.0",
-        "@typescript-eslint/parser": "~5.60.0",
+        "@typescript-eslint/eslint-plugin": "~5.62.0",
+        "@typescript-eslint/parser": "~5.62.0",
         "eslint-plugin-eslint-comments": "~3.2.0",
         "eslint-plugin-eslint-plugin": "~4.4.1",
         "eslint-plugin-node": "~11.1.0",
         "eslint-plugin-prettier": "~3.4.1",
         "eslint-plugin-vue": "~8.7.1",
         "prettier": "~2.8.8",
-        "semver": "^7.5.3",
+        "semver": "^7.5.4",
         "vue-eslint-parser": "^8.3.0"
     },
     "devDependencies": {
         "@eslint-community/eslint-plugin-mysticatea": "file:.",
-        "@eslint/eslintrc": "^2.0.3",
-        "eslint": "~8.43.0",
-        "eslint-config-prettier": "^8.8.0",
+        "@eslint/eslintrc": "^2.1.1",
+        "eslint": "~8.46.0",
+        "eslint-config-prettier": "^8.9.0",
         "globals": "^13.20.0",
         "mocha": "^9.2.2",
         "npm-run-all": "^4.1.5",
         "nyc": "^15.1.0",
         "opener": "^1.5.2",
         "rimraf": "^3.0.2",
-        "typescript": "^5.1.3"
+        "typescript": "^5.1.6"
```

TypeScript v5.1 has dropped support for node v12. So it seems we should use TypeScript v5.0.

```suggestion
        "typescript": "~5.0.4"
```


https://devblogs.microsoft.com/typescript/announcing-typescript-5-1/#breaking-changes